frontends/mit-learn/src/page-components/Dialogs/AddToListDialog.tsx

```diff
@@ -202,7 +200,7 @@ const AddToLearningPathDialogInner: React.FC<AddToListDialogProps> = ({
 }) => {
   const resourceQuery = useLearningResourcesDetail(resourceId)
   const resource = resourceQuery.data
-  const listsQuery = useLearningPathsList()
+  const listsQuery = useLearningPathsList({ limit: LIST_LIMIT })
```

Comment:
Showing up to 100 and scrolling isn't a great solution. We should have infinite scroll or something.

This is a remediation to unblock the DLLs.